csrc/lower2device.h

```diff
@@ -62,6 +62,10 @@ class TORCH_CUDA_CU_API GpuLower : public NonCopyable {
 
   kir::Kernel* kernel() const;
 
+  Fusion* fusion() const {
```

Comment:
Looks like this just returns the same pointer as the above `kernel()`.

Reply:
I wasn't aware of that. Thanks for pointing out!

Reply:
fixed